ROADMAP.md

```diff
@@ -0,0 +1,55 @@
+# Roadmap
+
+This document defines a high level roadmap for Rook development. The dates below are subject to change but should give a general idea of what we are planning. We use the milestone feature in Github so look there for the most up-to-date and issue plan.
+
+## Rook 0.5 (June 2017)
+
+| Platform   | Block  | Object | File   |
+|------------|--------|--------|--------|
+| Kubernetes | Alpha  | Alpha  | Alpha  |
+| Standalone | Alpha  | Alpha  | Alpha  |
+
+ - Rook still in alpha
+ - MON and OSD Reliability - survive restarts and pod failures
+ - Backup of the cluster config and data required to recover it
+ - Publish documentation to rook.io
+ - RBD works on all Kubernetes deployments (without any host changes)
+ - Upgrade to Ceph Luminous
+ - Long-haul testing infra
+ - TPRs for object store and file system
+ - Initial Rook Upgrade support
```

Comment:
since upgrade isn't coming until 0.6+ this is only the design, right? What if we remove it from 0.5 to be clear that there is no upgrade support yet?